Runtimes/Core/core/CMakeLists.txt

```diff
@@ -306,7 +306,8 @@ target_link_libraries(swiftCore
     swiftDemangling
     swiftShims
     swiftStdlibStubs
-    swiftThreading)
+    swiftThreading
+    $<$<NOT:$<PLATFORM_ID:Darwin>>:swiftrt$<$<PLATFORM_ID:Windows>:T>>)
```

Comment:
Uninformed question: why this is not written as `$<$<PLATFORM_ID:Windows>:swiftrtT>`? Is it because we need to link `swiftrt` on Linux and other platforms?

Reply:
Correct, currently, for all non-Darwin targets, we link against `swiftrt`. On Windows, we need to use `swiftrtT` for swiftCore but use `swiftrt` for everything else.